.github/workflows/release.yaml

```diff
@@ -86,3 +108,7 @@ jobs:
           docker image tag tracee:latest aquasec/tracee:${ARCH}-${TAG}
           docker image push aquasec/tracee:${ARCH}
           docker image push aquasec/tracee:${ARCH}-${TAG}
+      - name: Sign Docker image
+        run: |
+          cosign sign -y $(docker inspect --format='{{index .RepoDigests 0}}' aquasec/tracee:${ARCH})
```

Comment:
in addition to that, once you push the image, you need to re-pull it to be able to fill the digests field of the configuration of the image, instead, you can use the `crane` tool to find the digests without needing to re-pull the image.

```
cosign sign -y aquasec/tracee:${ARCH}@$(crane digest aquasec/tracee:${ARCH})
```

Reply:
this is also great! thanks, didn't know this! Otherwise, everything looks correct to me!